src/python/install.sh

```diff
@@ -218,13 +306,71 @@ add_symlink() {
     fi
 }
 
+install_openssl11() {
+    local _prefix=$1
+    mkdir /tmp/openssl11
+    (
+    cd /tmp/openssl11
+    curl -L -f -O https://www.openssl.org/source/openssl-1.1.1w.tar.gz
+    tar xzf openssl-1.1.1w.tar.gz
+    cd openssl-1.1.*
+    ./config --prefix=${_prefix} --openssldir=${_prefix}
+    make -j $(nproc)
+    make install_dev
```

Comment:
nit: Can we indent this by one tab? Thanks!

Reply:
Commit [baf1568](https://github.com/devcontainers/features/pull/830/commits/baf1568d80bb6f1eb73dd7e1b774b2fa5f3b662d) addresses this.